### DISCUSSION TITLE
Intranet

### DISCUSSION THREAD

**Discussion by brucebailey** [MasaCMS/MasaCMS#345] [Q&A]
Years ago there was a plugin that you could connect to active directory in order to use MASA as an intranet. Does that still exist somewhere and is up to date by chance?

**Comment by no3rdw** [comment]
This is a very late reply, but I think you're referring to the ['MuraUserBridge' plugin](https://github.com/blueriver/MuraUserBridge).  Blueriver had taken out the actual AD connection code because everyone's LDAP/AD setup is so different, but this still may be a useful starting point to implement something custom.  We still use this plugin on our company Intranet to automatically create Mura accounts and login users, although we're not using AD.

**Reply by brucebailey** [reply]
Yes, that is the guy, thank you! Trying to push this for a company intranet over other CMS options but having some push back because plugins for other platforms are readily available.

**Reply by guustnieuwenhuis** [reply]
Masa CMS also supports SSO with Microsoft Entra (AD on Azure).

**Reply by brucebailey** [reply]
I just saw this in the 7.5 release email that came! Pretty stoked on that.. thank you

Is there documentation on using this integration anywhere?
